.github/workflows/i18n_update.yaml

```diff
@@ -0,0 +1,54 @@
+name: i18n Updater
+# This Workflow will periodically go in: 
+# 1. update the i18n repo.
+# 2. file a pr. 
+# 3. approve the pr. 
+# 4. Mark is as auto merge, which means, if CI does pass, this will be merged automatically. 
+
+
+on:
+  # Triggers the workflow on pull request events but only for the main branch
+  # For now let's only have it manually, until we know this works! 
+  # schedule:
+  #  - cron: '0 5 * * *'
```

Comment:
Is there a ticket to validate the job and enable this? If there is, would you mind adding a link to it here?

Reply:
Ah i should have noted: i played with this on my fork. 
-> Task Run: https://github.com/strseb/mozilla-vpn-client/actions/runs/5135309815
-> Created and merged this PR: https://github.com/strseb/mozilla-vpn-client/pull/65 

But created VPN-4905 to test this and make it a cron once it works :)